.pipelines/templates/template-job-deploy-azure-env.yml

```diff
@@ -76,6 +78,8 @@ jobs:
         location: ${{ location }}
         subscription: ${{ parameters.e2eSubscription }}
         azureDevOpsE2EJSONSPN: ${{ parameters.azureDevOpsE2EJSONSPN }}
+        aroVersionStorageAccount: ${{ parameters.aroVersionStorageAccount }}
```

Comment:
all of the yaml in here from `- job: RP_E2E_${{ location }}` down is unwanted duplicate and should call into .pipelines/templates/template-rp-and-billing-e2e.yml.  Please open a KTLO VSTS to clean up.